test/e2e/performanceprofile/functests/README.txt

```diff
@@ -1,5 +1,16 @@
-HEADS UP!
+Functest consist in diffent suites:
```

Comment:
typo: different
Alternative phrasing:
`The functional tests (functests) are split across suites as follows:`